README.adoc

```diff
@@ -56,6 +56,16 @@ installation)
 
     cert-magic tidy -d www.example.com
 
+== Checking certificates
+
+Once a certificate has been installed, check it using:
+
+https://www.ssllabs.com/ssltest/
```

Comment:
I'd make this an actual link inline for convenience, but also so it takes up a bit less space.

------------------------

## Checking certificates

Once a certificate has been installed, check it using [the SSL Labs server test](https://www.ssllabs.com/ssltest/), making sure 'Do not show the results on the boards' is checked.

Anything less than an 'A' grade is worth investigating.